content/en/blog/performance-benchmarks-gke.md

```diff
@@ -0,0 +1,354 @@
+      integration is set to 5s.
+
+   1. The load on the controller is reduced, since there are fewer LoadTest
+      resources at a time for it to control.
+
+   1. Each test can have a shorter timeout period, since the time takes for the
+      controller to start each test is more predictable. Timeouts are necessary
+      to account for error cases where client or server pods hang and prevent
+      the test from completing. These cases are rare but can accumulate and
+      consume cluster resources, preventing other tests from running. Tests in
+      our continuous integration have a timeout of 15 minutes.
+
+   1. The concurrency level can be set lower than the capacity of the cluster,
+      allowing a user to run a batch of tests without preventing other users
+      from running tests simultaneously.
+
+1. The option of deleting each test once it completes successfully (and after
+   results and logs are collected) provides better control of the lifecycle of
+   each test.
+
+   1. The default behavior of the controller is to keep a LoadTest resource and
+      associated pods on the cluster until it reaches a set TTL, and then delete
+      it. Our continuous integration specifies a TTL of 24h for each test.
+
+   1. Pods belonging to a completed LoadTest are in terminated state, and so
+      consume no resources on the cluster. However, terminated pods can be
+      garbage-collected at any time.
+
+   1. If we let pods belonging to all completed tests stay in our continuous
+      integration cluster, we find that they are garbage-collected within one
+      hour.
+
+   1. If we delete LoadTest resources for tests that complete successfully, the
+      associated pods are also deleted. In this case, the pods belonging to
+      _unsuccessful_ tests, which are few in number and may be useful for
+      debugging, stay on the cluster until the 24h TTL is reached.
+
+[test runner]:
+  https://github.com/grpc/test-infra/blob/master/tools/README.md#test-runner
+
+### Dashboard
+
+Test results from continuous integration are saved to [BigQuery]. The data
+stored in BigQuery is then replicated to a Postgres database for visualization
+on a dashboard.
+
+The code for the dashboard, as well as the configuration of the main continuous
+integration dashboard, are stored in the [Test Infra] repository. This brings
+the following benefits:
+
+1. The main dashboard is maintained by updating the stored configuration,
+   instead of updating it directly in the UI.
+
+1. Users can deploy their own dashboard, using their own configuration.
+
+This contrasts with the dashboard for the previous benchmarks, built using
+Perfkit Explorer, which was maintained by updating it directly in the UI, and
+could not be easily replicated by users.
+
+For details, see [dashboard implementation].
+
+Dashboard snapshot:
+
+[![Dashboard snapshot]](https://grafana-dot-grpc-testing.appspot.com)
```

Comment:
Add "<span class="hk-no-external-icon"></span>" to the link text field, like this:

    [![Dashboard snapshot]<span class="hk-no-external-icon"></span>](https://grafana-dot-grpc-testing.appspot.com)

Reply:
Done.